_source/user-guide/integrations/custom-endpoints.md

```diff
@@ -75,6 +75,43 @@ Some endpoints require a particular attribute/payload. For example, [Microsoft T
 ![Configure a custom endpoint](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/custom-endpoint-POST.png)
 
 
+##### Configure your payload to add data to your alert notifications
+
+You can enhance your alert notifications with data pulled from the triggered alert.
+
+For example, you can add a **View in Kibana** drilldown link to a notification using the following:
+
+```
+"alert_view_link": "{{&alert_app_url}}#/view-triggered-alert?from={{&alert_timeframe_start_epoch_millis}}&to={{&alert_timeframe_end_epoch_millis}}&definitionId={{&alert_definition_id}}&switchToAccountId={{&account_id}}"
+```
+
+Logz.io notifications use [Mustache templating](https://mustache.github.io/).
+All variables are HTML escaped by default. If you want to return **unescaped HTML**, add an **ampersand (&)**, for example `{{&alert_title}}`.
+
+{% raw %}
+
+| Parameter | Description |
+|---|---|---|
+| {{alert_title}} | Title of the alert that triggered  |
+| {{alert_description}} | Description of the alert|
+| {{alert_definition_id}} | Unique alert ID. |
+| {{alert_severity}} | Severity of the alert that triggered  |
+| {{account_id}} | ID of the affected Logz.io account  |
+| {{account_name}} |  Name of the affected Logz.io account |
+| {{alert_samples}} | Prints a sample of the raw logs that caused the alert to trigger. (JSON format, max. of 10 logs.) |
+| {{alert_event_html}} | Returns an HTML file with the output table, as defined in the alert. Only relevant if the alert is configured to output data in tables. |
+| {{alert_timeframe_start}} | Start time for the triggered alert event.   |
+| {{alert_timeframe_end}} | End time for the triggered alert event  |
+| {{alert_severity_img}} | Colored icon indicating the alert severity  |
+| {{alert_timeframe_start_epoch_millis}} |  Start time for the triggered alert event in UNIX milliseconds |
+| {{alert_timeframe_end_epoch_millis}} |  End time for the triggered alert event in UNIX milliseconds |
+| {{alert_app_url}}| The domain to your Logz.io account, for example: `https://app.logz.io`. Is used to build links. |
+
```

Comment:
{{alert_event_image}} | Returns a URL to an image file with the output table, as defined in the alert. Only relevant if the alert is configured to output data in tables.